pipes/WDL/workflows/demux_metag.wdl

```diff
@@ -5,18 +5,26 @@ import "tasks_metagenomics.wdl" as metagenomics
 import "tasks_taxon_filter.wdl" as taxon_filter
```

Comment:
If you think this is ready and want to try it out, shouldn't you remove #DX_SKIP_WORKFLOW?